### ISSUE TITLE
Eliminate `localstack` from the code.

### ISSUE THREAD

**Issue by MathieuDutSik** [linera-io/linera-protocol#932]
We used to use localstack for testing the `DynamoDb` functionality and so the code is littered with its mention.
However, we are no longer using it so this should be cleaned up.

**Comment by ashu26jha** [comment]
Hello! I think it would be better to discuss my approach before making a PR

1. In `linera-views` I am planning to eliminate localstack from `mod.rs`, `views.rs`, `operations_tests.rs` and `views_test.rs`. Also making sure `S3` is not impacted/modified. (I need clarity whether the localstack should be removed from entire code-base or only where it is used to test DynamoDB)

2. I won't cram all the changes in a single PR, rather a separate PR for `linera-views`, `linera-core`, `linera-service` etc to ease up the reviewing process

**Comment by ma2bd** [comment]
@ashu26jha Despite the (slightly confusing) title, I suspect this task is more about renaming stuff because we switched from LocalStack to DynamoDbLocal:
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html

@MathieuDutSik Did I miss something? (And considering clarifying the task)

**Comment by MathieuDutSik** [comment]
Yes, making sense of what can be simplified is part of the challenge of this issue.
Currently, the word `localstack` occurs 102 times in the codebase, which feels abnormal for some functionality that we do not use. Maybe this is not a "first issue".

**Comment by ma2bd** [comment]
I looked at the code and there is nothing much to do except renaming things.
